public/js/graph/parse_graph.js

```diff
@@ -45,7 +45,9 @@ function buildGraph() {
     });
 
     $('.path').each(function () {
-        makeEdge(window[$(this).attr('source-node')], window[$(this).attr('target-node')], $(this).attr('id'));
+        makeEdge(window[$(this).attr('source-node').replace(',', '')],
```

Comment:
Done.

The server implementation of this can be simplified, I'll add it to my refactoring list. (Filtering commas once, rather than three times).